{{cookiecutter.project_slug}}/MANIFEST.in

```diff
@@ -1,3 +1,4 @@
+graft {{ cookiecutter.project_slug }}
```

Comment:
@eliasdorneles
I dont no. I can't find any docs about Manifest.in file.

maybe change all to use the same way of django

https://github.com/django/django/blob/master/MANIFEST.in